examples/internal_consent_example.js

```diff
@@ -19,7 +22,7 @@ Countly.add_event({
     }
 });
 
-//add internal events to see if they get their respective consents instead of event consent
+//add internal events to see if they get their "respective consents" instead of "event" consent
```

Comment:
Before this line remove "events" consent and add other consents for internal events